batch/batch/server/server.py

```diff
@@ -763,36 +812,54 @@ def update_job_with_pod(job, pod):
         job.mark_unscheduled()
 
 
-@routes.post('/pod_changed')
-async def pod_changed(request):
-    parameters = await request.json()
+@routes.get('/recent')
+@aiohttp_jinja2.template('recent.html')
+@authenticated_users_only
+async def recent(request):  # pylint: disable=W0613
+    recent_events = get_recent_events()
+    return {'recent': list(reversed(recent_events))}
 
-    pod_name = parameters['pod_name']
 
-    job = pod_name_job.get(pod_name)
+def blocking_to_async(f, *args, **kwargs):
+    return asyncio.get_event_loop().run_in_executor(
```

Comment:
It is recommended to use `get_running_loop`: [docs](https://docs.python.org/3/library/asyncio-eventloop.html) where possible. In that case, you could run `asyncio.set_event_loop(asyncio.new_event_loop())` within `serve`, and `get_running_loop` elsewhere.

Minor

Reply:
thats py 3.7 only and we're stuck on py 3.6